src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -150,23 +156,34 @@ class SelectedFilesDownloadModal extends React.PureComponent {
         };
     }
 
-    handleAcceptDisclaimer(){
-        const { context } = this.props;
-        const { filters = null, display_title, title } = context || {};
-        const eventObj = {
-            name: display_title || title
-        };
-
-        if (filters) {
-            eventObj.currentFilters = analytics.getStringifiedCurrentFilters(filters);
+    componentDidMount(){
+        const { analyticsAddFilesToCart = false, fileCountUnique, selectedFiles = {}, context } = this.props;
+        if (!analyticsAddFilesToCart){
+            return;
         }
 
-        analytics.event("SelectedFilesDownloadModal", "Accepted Disclaimer", eventObj);
+        const fileList = _.keys(selectedFiles).map(function(accessionTripleString){
+            return selectedFiles[accessionTripleString];
+        });
+        const extData = { list: analytics.hrefToListName(window && window.location.href) };
+        analytics.productsAddToCart(fileList, extData);
+        analytics.event("SelectedFilesDownloadModal", "Mounted", {
+            ...analytics.eventObjectFromCtx(context),
+            eventValue: fileCountUnique || fileList.length || 0
+        });
+    }
+
+    handleAcceptDisclaimer(){
+        const { context, fileCountUnique } = this.props;
+        analytics.event("SelectedFilesDownloadModal", "Accepted Disclaimer", {
+            ...analytics.eventObjectFromCtx(context),
+            eventValue: fileCountUnique
+        });
         this.setState({ 'disclaimerAccepted' : true });
     }
 
     render(){
-        const { onHide, filenamePrefix, selectedFiles, fileCountUnique, fileCountWithDupes } = this.props;
+        const { onHide, filenamePrefix, selectedFiles, fileCountUnique, fileCountWithDupes, context } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['fileCountWithDupes' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)